config/jobs/kubernetes/sig-cluster-lifecycle/k8s-upgrade-gke.yaml

```diff
@@ -1,665 +0,0 @@
-      - --timeout=900m
-      - --upgrade_args=--ginkgo.focus=\[Feature:ClusterDowngrade\] --upgrade-target=ci/k8s-stable2 --upgrade-image=gci
-      image: gcr.io/k8s-testimages/kubekins-e2e:v20190906-fc85258-master
-
-  annotations:
-    testgrid-dashboards: sig-cluster-lifecycle-upgrade-skew
-    testgrid-tab-name: gke-gci-1.13-gci-1.12-downgrade-cluster
-- interval: 12h
-  name: ci-kubernetes-e2e-gke-gci-stable1-gci-stable2-downgrade-cluster-parallel
-  labels:
-    preset-service-account: "true"
-    preset-k8s-ssh: "true"
-  spec:
-    containers:
-    - args:
-      - --timeout=140
-      - --bare
-      - --scenario=kubernetes_e2e
-      - --
-      - --check-leaked-resources
-      - --check-version-skew=false
-      - --deployment=gke
-      - --extract=ci/k8s-stable2
-      - --extract=ci/k8s-stable1
-      - --gcp-cloud-sdk=gs://cloud-sdk-testing/ci/staging
-      - --gcp-node-image=gci
-      - --gcp-zone=us-central1-c
-      - --ginkgo-parallel
-      - --gke-environment=test
-      - --provider=gke
-      - --skew
-      - --test_args=--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\] --kubectl-path=../../../../kubernetes_skew/cluster/kubectl.sh --minStartupPods=8
-      - --timeout=120m
-      - --upgrade_args=--ginkgo.focus=\[Feature:ClusterDowngrade\] --upgrade-target=ci/k8s-stable2 --upgrade-image=gci
-      image: gcr.io/k8s-testimages/kubekins-e2e:v20190906-fc85258-master
-
-  annotations:
-    testgrid-dashboards: sig-cluster-lifecycle-upgrade-skew
-    testgrid-tab-name: gke-gci-1.13-gci-1.12-downgrade-cluster-parallel
-- interval: 12h
-  name: ci-kubernetes-e2e-gke-gci-stable2-gci-stable1-upgrade-cluster
-  labels:
-    preset-service-account: "true"
-    preset-k8s-ssh: "true"
-  spec:
-    containers:
-    - args:
-      - --timeout=920
-      - --bare
-      - --scenario=kubernetes_e2e
-      - --
-      - --check-leaked-resources
-      - --check-version-skew=false
-      - --deployment=gke
-      - --extract=ci/k8s-stable1
-      - --extract=ci/k8s-stable2
-      - --gcp-cloud-sdk=gs://cloud-sdk-testing/ci/staging
-      - --gcp-node-image=gci
-      - --gcp-zone=us-central1-c
-      - --gke-create-command=container clusters create --quiet --enable-legacy-authorization
-      - --gke-environment=test
-      - --provider=gke
-      - --test_args=--kubectl-path=../../../../kubernetes_skew/cluster/kubectl.sh --minStartupPods=8
-      - --timeout=900m
-      - --upgrade_args=--ginkgo.focus=\[Feature:ClusterUpgrade\] --upgrade-target=ci/k8s-stable1 --upgrade-image=gci
-      image: gcr.io/k8s-testimages/kubekins-e2e:v20190906-fc85258-master
-
-  annotations:
-    testgrid-dashboards: sig-release-1.13-all, sig-cluster-lifecycle-upgrade-skew, sig-gcp-release-1.11
-    testgrid-tab-name: gke-gci-1.12-gci-1.13-upgrade-cluster
-- interval: 12h
-  name: ci-kubernetes-e2e-gke-gci-stable2-gci-stable1-upgrade-cluster-new
```

Comment:
also on the verge of being fixable https://testgrid.k8s.io/sig-cluster-lifecycle-upgrade-skew#gke-gci-1.12-gci-1.13-upgrade-cluster-new